packages/web-app-admin-settings/src/views/Spaces.vue

```diff
@@ -205,34 +235,51 @@ export default defineComponent({
       ].filter((p) => p.enabled)
     })
 
+    const limitedScreenSpace = ref(false)
+    const appBarActionsRef = ref()
+    const onResize = () => {
+      limitedScreenSpace.value = unref(sideBarOpen)
+        ? window.innerWidth <= 1280
+        : window.innerWidth <= 1000
+    }
+    const resizeObserver = new ResizeObserver(onResize)
+
     onMounted(async () => {
       await loadResourcesTask.perform()
       loadResourcesEventToken.value = eventBus.subscribe('app.admin-settings.list.load', () => {
         loadResourcesTask.perform()
+        selectedSpaces.value = []
       })
 
       calculateListHeaderPosition()
       window.addEventListener('resize', calculateListHeaderPosition)
+      unref(resizeObserver).observe(unref(appBarActionsRef))
     })
 
     onBeforeUnmount(() => {
       eventBus.unsubscribe('app.admin-settings.list.load', unref(loadResourcesEventToken))
+      unref(resizeObserver).unobserve(unref(appBarActionsRef))
```

Comment:
resize observer is not a ref anymore ;-)

Reply:
Whoops 😅